gitter.py

```diff
@@ -149,11 +149,12 @@ class GitterRoom(Room):
     def invite(self, *args) -> None:
         pass
 
-    def __init__(self, backend, idd, uri, name):
+    def __init__(self, backend, idd, uri, name, topic=None):
         self._backend = backend
         self._name = name
         self._uri = uri
         self._idd = idd
+        self._topic = topic
```

Comment:
We're not making use of the `topic` parameter nor `self._topic` anymore, no need to store it.